### PR TITLE
Ostrich Pull Request

### PR DESCRIPTION
**Pull request by SCOU7**
two instances of Ostrich were added to the main method of Trader.java.
class Ostrich.java and test class OstrichTest.java created.

### CODE REVIEW

File: src/main/java/Trader.java [hunk 1]
```diff
@@ -96,8 +96,9 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
```
Comment (spalmurray) [inline src/main/java/Trader.java]: need comma after list item. Also, remove TODO.

File: src/test/java/OstrichTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/*
+ * This file contains sample JUnit test cases for Ostrich.java
+ */
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+
+public class OstrichTest {
```
Comment (spalmurray) [inline src/test/java/OstrichTest.java:10]: Need test cases for downgrade speed and toString.

**Review by spalmurray — CHANGES_REQUESTED** [review]

File: src/test/java/OstrichTest.java (hunk 2 above)
Comment (SCOU7) [reply src/test/java/OstrichTest.java:10]: test case for downgrade speed added.

**Review by SCOU7 — COMMENTED** [review]
All required changes - adding test case for downgrade speed - made.

**Pull request merged**